docker/stellargraph/Dockerfile

```diff
@@ -1,14 +1,18 @@
+FROM python:3.6-slim as builder
+
+ENV PATH=${PATH}:/root/.local/bin
+WORKDIR /build/
+
+COPY . /build/
+
+RUN pip install --no-cache-dir -r /build/requirements.txt /build/ --user
+
 FROM python:3.6-slim
 
 RUN adduser --disabled-password --gecos '' stellar
 
+COPY --chown=stellar --from=builder /root/.local /home/stellar/.local
```

Comment:
Could you add a comment or two about how this all works? E.g. referencing that `--user` will install to `.local`?